Geometry/EcalAlgo/plugins/BuildFile.xml

```diff
@@ -7,6 +7,6 @@
   <flags EDM_PLUGIN="1"/>
 </library>
 
-<library file="Ecal*.cc moduleDD4Hep.cc" name="GeometryEcalAlgoDD4HepPlugin">
+<library file="Ecal*.cc moduleDD4hep.cc" name="GeometryEcalAlgoDD4hepPlugin">
```

Comment:
Should the file `moduleDD4Hep.cc` in `Geometry/EcalAlgo/plugins` have its name changed to `moduleDD4hep.cc`?

Reply:
done in https://github.com/cms-sw/cmssw/pull/36021